packages/webview_flutter/android/src/main/java/io/flutter/plugins/webviewflutter/FlutterWebView.java

```diff
@@ -73,6 +75,23 @@ public boolean shouldOverrideUrlLoading(WebView view, String url) {
       return true;
     }
 
+    @Override
+    public boolean onShowFileChooser(
+        WebView webView,
+        ValueCallback<Uri[]> filePathCallback,
+        FileChooserParams fileChooserParams) {
+      // info as of 2021-03-08:
+      // don't use fileChooserParams.getTitle() as it is (always? on Mi 9T Pro Android 10 at least) null
+      // don't use fileChooserParams.isCaptureEnabled() as it is (always? on Mi 9T Pro Android 10 at least) false, even when the file upload allows images or any file
+      final Context context = webView.getContext();
+      final boolean allowMultipleFiles =
+          fileChooserParams.getMode() == FileChooserParams.MODE_OPEN_MULTIPLE;
+      new FileChooserLauncher(
+              context, allowMultipleFiles, filePathCallback, fileChooserParams.getAcceptTypes())
+          .start();
```

Comment:
Android >= 5.0

I guess it should be something like this:
```java
   final boolean allowMultipleFiles = Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP
      && fileChooserParams.getMode() == FileChooserParams.MODE_OPEN_MULTIPLE;
   final String[] acceptTypes = Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP
      ? fileChooserParams.getAcceptTypes() : new String[0];
   new FileChooserLauncher(context, allowMultipleFiles, filePathCallback, acceptTypes)
      .start();
```

Reply:
Thanks for your comment. I've added this change.